submissions/Khatsur/html-css-popup/style.css

```diff
@@ -0,0 +1,116 @@
+body {
+  font-family: Arial, sans-serif;
+}
+
+nav {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.navlist {
+  display: flex;
+  width: auto;
+  justify-content: flex-end;
+  align-items: center;
+  flex-wrap: wrap;
+  list-style-type: none;
+}
+
+li {
+  margin: 20px;
+  cursor: pointer;
+}
+
+li:hover {
+  border-radius: 5px;
+}
+
+li:active {
+  border: 1px solid blue;
+}
+
+.popups {
+  border-radius: 5px;
+  position: absolute;
+  right: 49px;
+  width: 340px;
+}
+
+.iconsarea {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  border: 1px solid black;
+  padding: 20px;
+  background: #fff;
+  margin-top: 50px;
+  width: 100%;
+  height: 220px;
+  overflow: hidden;
+  box-shadow: rgba(100, 100, 111, 0.2) 0px 7px 29px 0px;
+}
+
+.iconsarea:target {
+  overflow: scroll;
+  height: 350px;
+}
+
+img.popimg {
```

Comment:
Avoid using too specific CSS selectors. [Read more about that](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)